### ISSUE TITLE
Installation intructions unclear

### ISSUE THREAD

**Issue by ejm554** [packwiz/packwiz#160]
Hello. I'm looking forward to trying packwiz. Unfortunately, the installation instructions are unclear -- at least to me. They say:

> _Prebuilt binaries are available from [GitHub Actions](https://github.com/packwiz/packwiz/actions) - the UI is a bit terrible, but essentially select the top build, then download the artifact ZIP for your system at the bottom of the page._

### I don't know what is meant by the "top build" in GitHub Actions.

When I open Actions, this is what I see:

![image](https://user-images.githubusercontent.com/26723535/197919104-72a7cabc-61ad-4197-b111-81154a45f566.png)

- I see a lot of "workflow runs." Do you mean that I should select the top workflow? 
    - I did select and open it, but the results didn't look like something that I should download.

- On the Actions page, I even searched for the word "build," but it doesn't appear on the page. 

### I also don't know what is meant by "download the artifact ZIP for your system at the bottom of the page."

- I looked at the bottom of the page, and I didn't find any ZIP files.
- I only see page numbers, presumably to see more workflows.

![image](https://user-images.githubusercontent.com/26723535/197921282-08f34e7a-085e-4611-bea6-d6707e2f3309.png)

Admittedly, I could be missing something obvious, and your installation instructions make sense to most people, but I'm unsure how to proceed. Is some assistance possible? On the other hand, if you agree that your instructions could be clarified in some way, perhaps someone could revise them. :-)

**Comment by comp500** [comment]
Ah hmm, this made more sense before the nix workflows were added (which don't produce the binaries). You'll want to select the *second* workflow run (`Go #130: Commit f00dc98`), and grab one of the artifacts at the bottom of that page.

I want to move away from using Github Actions for releases entirely at some point, it's rather confusing :P

**Comment by ryleu** [comment]
wouldn't running `go run github.com/packwiz/packwiz@latest` install it properly? it did on my machines

**Comment by comp500** [comment]
> wouldn't running `go run github.com/packwiz/packwiz@latest` install it properly? it did on my machines

Yes, but that requires having a Go install (unnecessary since I can easily compile binaries in CI) and that the Go install is set up properly (way too many distro packages don't add go-installed binaries to the PATH). I do also want to add a built-in update check mechanism, since people don't tend to regularly check for packwiz updates.
